.travis.yml

```diff
@@ -13,5 +13,8 @@ matrix:
   allow_failures:
     - node_js: "node"
 
+before_script:
+  - export MOCHA_OPTS=$(node --v8-options | grep --only-matching -- '--harmony_generators')
+
 after_script:
-  make coveralls
+  npm run coveralls
```

Comment:
I don't think these commands are equivalent. The Makefile was set to run Istanbul first, whereas this just runs coveralls.